server/chart_config.json

```diff
@@ -419,6 +442,7 @@
     "category": "Health",
     "titleId": "CHART_TITLE-Mortality_events",
     "title": "Mortality events",
+    "description": "Count of deaths caused by various diseases",
```

Comment:
Nit: I think these mortality events are not necessarily due to disease. Some are external circumstances

Reply:
thanks! updated